docs/source/data-types/udt.md

```diff
@@ -17,16 +40,12 @@ use scylla::IntoTypedRows;
 use scylla::macros::{FromUserType, IntoUserType};
 use scylla::cql_to_rust::FromCqlVal;
 
-// Define custom struct that matches User Defined Type created earlier
-// wrapping field in Option will gracefully handle null field values
```

Comment:
I'm not sure why you removed the mention of `Option` handling null values gracefully.